Unity/Assets/ClassSelectionAvailableExample.cs

```diff
@@ -22,28 +46,23 @@ void OnApplicationQuit()
     void Update()
     {
         //TODO ADD YOUR CODE HERE
-        if(_update)
+        if (_update)
         {
-            switch (_selectedClass)
+            if (_selectedClass != 0)
```

Comment:
We need to check that it is between 1 and 4

Reply:
ok